RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
 = Releasing Kiali
 
-This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
+This document covers the specifics of releasing Kiali, and is meant as a guide
+for Kiali release managers.
+
+The summary of the process is the following:
+
+* Create a new JIRA issue for the release
+* Make sure Getting Started docs are complete
+* Update Kiali UI version
```

Comment:
link to where and how to update?

Reply:
It's on the section, there's a link there to the kiali-ui release steps (which do not exist yet per https://github.com/kiali/kiali-ui/pull/516).